ex7.13_NumPy_convert_2_1D_to_1_2D.py

```diff
@@ -0,0 +1,18 @@
+import numpy
+import numpy as np
+
+
+def convert_two_1D_to_one_2D_matrix(first_matrix: numpy.ndarray, sec_matrix: numpy.ndarray):
+    """
+    function that convert (in sequence depth wise (along third axis)) two 1-D arrays into a 2-D array.
```

Comment:
Capitalize, *converts, brackets inside brakets is not a common thing to do in English.